tests/nightly/test_large_array.py

```diff
@@ -467,6 +468,20 @@ def check_col2im():
         assert res.shape[2] == 2
         assert res.shape[3] == 2
         assert res.shape[4] == 1
+        
+    def check_multi_lars():
+        lrs = nd.random_normal(shape=(LARGE_TENSOR_SHAPE + 1, 1))
+        weights_sum_sq = nd.random_normal(shape=(LARGE_TENSOR_SHAPE + 1, 1))
+        grads_sum_sq = nd.random_normal(shape=(LARGE_TENSOR_SHAPE + 1, 1))
+        wds = nd.random_normal(shape=(LARGE_TENSOR_SHAPE + 1, 1))
+        eta = .1
+        eps = .9
+
+        out = nd.multi_lars(lrs=lrs, weights_sum_sq=weights_sum_sq, grads_sum_sq=grads_sum_sq,
+                            wds=wds, eta=eta, eps=eps)
+
```

Comment:
shape assertions don't necessarily mean the NDArrays would be created right?
let's check for values as well

Reply:
@connorgoggins just check few selected values. No need to check all since tensor size is >5B

Reply:
Great observation - updated w/type check on the first element to trigger evaluation of the output NDArray.